facebook.py

```diff
@@ -46,15 +46,11 @@
 import socket
 
 # Find a JSON parser
-try:
-    import simplejson as json
-except ImportError:
-    try:
-        from django.utils import simplejson as json
-    except ImportError:
-        import json
+import simplejson as json
```

Comment:
If you do this make sure to update the requirements/setup.py whatever to make sure simplejson is listed as a dependency. @tudor-mihordea-hs